packit_service/worker/handlers/copr.py

```diff
@@ -275,12 +276,22 @@ def run(self):
             logger.info(msg)
             return TaskResults(success=True, details={"msg": msg})
 
+        self.pushgateway.copr_builds_finished.inc()
+
+        # if the build is needed only for test, it doesn't have the task_accepted_time
```

Comment:
so we don't take in account Copr builds, right?

also related question
> How are we going to handle the length of test run if the Copr build needs to be done?

imo it will basically track the time of running the whole pipelline, which is intended? or not?

Reply:
> so we don't take in account Copr builds, right?

when do you mean this?

> How are we going to handle the length of test run if the Copr build needs to be done?

we agreed with Tomas on measuring the time from when the test run is submitted so that we don't include the time for building in Copr here

Reply:
it's definitely worth considering but from user PoV - if they are not interested in actual RPM builds let's not account for them in the build-specific metrics

+1 wrt tests

I'm really excited about finally starting to see these numbers and charts: once we have those, we can start tinkering with the actual measurements to get numbers and charts which have the biggest use for us

Reply:
@lbarcziova sorry, I just realized I wrote like an idiot :D I was going back and forth between the commits and it merged for me…

1. OK, Copr build is ignored if needed for TF. Got it.
2. I just checked TF helper, it creates the entry **when actually** submitting tests to TF, so my question is irrelevant. :)